std_msgs/Header header
string id
string manufacturer
string model
string serial_num
string firmware_version
int32 num_configurations
string encoding
AreaLimits overscan
AreaLimits effective
ChipInfo chip
uint32 bayer_format
bool is_color
bool is_cool
bool has_bin1x1_mode
bool has_bin2x2_mode
bool has_bin3x3_mode
bool has_bin4x4_mode
ParamLimits gain_limits
ParamLimits offset_limits
ParamLimits usb_traffic_limits
ParamLimits red_wb_limits
ParamLimits green_wb_limits
ParamLimits blue_wb_limits
ParamLimits temperature_limits
int32 frame_width
int32 frame_height
float32 fps
builtin_interfaces/Time initial_connection
builtin_interfaces/Time last_connection